src/OnboardingSPA/components/CardWithOptions/index.js

```diff
@@ -1,7 +1,8 @@
+import classNames from 'classnames';
 import { memo } from '@wordpress/element';
 import { Icon, chevronRight } from '@wordpress/icons';
 
-const CardWithOptions = ( { title, options, skip, callback } ) => {
+const CardWithOptions = ( { title, options, skip, selection, callback } ) => {
```

Comment:
We could consider renaming `CardWithOptions` to something like `OptionWithTitleAndDescription` and pass `isSelected` directly to each specific option instead of having a generic `CardWithOptions` component that iterates over `options` in a particular format. This approach would provide us with better control over the component and enhance its reusability.

So in this case, the responsibility of looping over options and rendering them could be moved to the experience step or another appropriate card component, separating them and making each component more focused on its specific task.

Reply:
Ohhh Can you tell me a bit more about this, Didn't understand completely as I believe those individual options won't be reusable as they will have a specific styling to them tightly coupled and we would need the outer box styling wherever we reuse it.